docs/content/getting-started.md

```diff
@@ -57,6 +57,8 @@ you should adjust to your own environment.
           --region $REGION
         ```
 
+* HyperShift supports the Agent platform via [cluster-api-provider-agent](https://github.com/openshift/cluster-api-provider-agent). It requires the Infrastructure Operator to be installed. See instructions [here](https://github.com/openshift/assisted-service/blob/master/docs/user-guide/infrastructure-operator-olm.md).
+
```

Comment:
This will only work for GA builds I guess, we'll need a different approach for testing with pre-release CI/nightly builds (as is the case with other content delivered via OLM) - I wonder if it's worth breaking this into a new doc where we can cover the end-user OLM case, and the environment needed for developer testing?

Reply:
For CI/nightly builds we use the [assisted-installer image stream](https://github.com/openshift/release/blob/f6e321b7943a22682fa8e5cc6dfb49b5cc674e7e/ci-operator/config/openshift/cluster-api-provider-agent/openshift-cluster-api-provider-agent-master.yaml#L14)
I agree that we should add e2e instructions